extensions/classify/config.proto

```diff
@@ -0,0 +1,158 @@
+/* Copyright 2020 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package classify;
+
+
+// Configuration to classify http traffic.
+// Traffic classification can be done on mutiple dimensions using request and
+// response headers and other measurements. 
+// The plugin must be configured as request, response or log based on
```

Comment:
There is no response based plugins as far as I know. All plugins implement the same callbacks, except for access log based ones.

Reply:
Lines 25:26 is quite hard to follow, may be add this:

This filter generates attributes based on the configured classification and request/response being processed. These attributes can be consumed by downstream filters in the chain. For e.g. Stats plugin uses `operation` dimension produced by the classification plugin, then it must be executed in LOG phase *before* the Stats plugin.